extensions/amp-story-interactive/0.1/amp-story-interactive-slider.js

```diff
@@ -115,17 +119,68 @@ export class AmpStoryInteractiveSlider extends AmpStoryInteractive {
       (event) => event.stopPropagation(),
       true
     );
+
+    this.storeService_.subscribe(
+      StateProperty.CURRENT_PAGE_ID,
+      (currPageId) => {
+        if (currPageId != this.getPageEl().getAttribute('id')) {
+          return;
+        }
+        const DURATION_MS = 1500;
+        let startTime;
+        const animateFrame = (currTime) => {
+          // Set current startTime if not defined.
+          if (!startTime) {
+            startTime = currTime;
+          }
+          const elapsed = currTime - startTime;
+          if (elapsed < DURATION_MS) {
+            // Value between 0 and 1;
+            const timePercentage = elapsed / DURATION_MS;
+            // example with Math.sin animation
+            // Converts oscillation between 1 and -1, to 0 and 30.
+            const val = map(
+              Math.cos(timePercentage * Math.PI * 2),
+              1,
+              -1,
+              0,
+              30
+            );
```

Comment:
lets use in-out-quad for this. It could be something like:
```
const val = timePercentage < .5 
  ? map(easeInOutQuad(timePercentage * 2), 0, .5, 0, 15) 
  : map(easeInOutQuad((timePercentage - .5) * 2), .5, 1, 15, 0);
};
function easeInOutQuad (t) { return t<.5 ? 8*t*t*t*t : 1-8*(--t)*t*t*t}
```

[Codepen](https://codepen.io/philipbell/pen/yLbvPJm?editors=1010) for reference